### DISCUSSION TITLE
Question about DRM

### DISCUSSION THREAD

**Discussion by MinePlayersPE** [iptv-org/community#257] [Q&A] [answered]
Hello, while doing unrelated work I noticed that all of the [all MNCNow streams in this repo are Widevine protected](https://github.com/iptv-org/iptv/search?q=mncnow) (you can easily tell from the _vmx suffix, and also how ffmpeg complains about SAMPLE_AES if you replace them with hls instead)
Getting the license server is surprisingly easy though (there are loads of examples even within GitHub, and you can even get it yourself without any kind of subscriptions nor logins), so I'm not sure whether to try remove them or to keep them and attach the license servers (via #KODIPROPS? I'm actually not sure) just for completeness

**Comment by Dum4G** [comment]
I can asume it cuts off a lot of unwanted auditory and leaves a tech-savvy minority that is neglectable. We also have links that won't work without proper http refferrer and user agent applied

**Reply by MinePlayersPE** [reply]
Sorry, but I can't tell if that's a yes or a no
>it cuts off a lot of unwanted auditory

Can you elaborate on what this means?
>leaves a tech-savvy minority that is neglectable

I'm not actually sure which IPTV clients have DRM support outside of Kodi, but considering the rarity I'm pretty sure that the people that can run DRM is more likely to be in the minority
>We also have links that won't work without proper http referrer and user agent applied

True, but like before, DRM support is incredibly uncommon compared to custom headers

**Reply by Dum4G** [reply]
> Can you elaborate on what this means?

Nvm, just polemics. In fact, that's a good question since we usually stand for ability to play content on all platforms and mpd streams have very limited use. Let's wait for @freearhey response on this

**Comment by freearhey** [comment, marked as answer]
Any stream must work in VLC player, otherwise such links should be removed from the playlist.

**Comment by Dum4G** [comment]
@Tribalize Here's an answer related to mpd files. Basically we remove all mpd files unless VLC 4.0 would be able to play them
